examples/simple/commands.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+num_permutations=1000
+num_cores=4
+
+# Preprocess mutations.
+python ../../process_mutations.py \
+    -m  adjacency_list.tsv \
+    -ct NA \
+    -o  data.json
+
+# Compute mutation probabilities.
+python ../../compute_mutation_probabilities.py \
```

Comment:
Can we provide a random seed so the results are reproducible?

Reply:
We already provide a random seed (the index for the each permutation), so the results will be reproducible on the same machine, but they will not necessarily be consistent from machine to machine.

Reply:
Yes, the random seeds for the `n` permuted matrices are `i`, `i+1`, ..., `i+n-1`, where `i` is user-definable (the argument `start_index`) but 1 by default.  We can add an optional `seed` argument and use that to generate seeds for each of the permuted matrices, which is less opaque but similar enough in practice.  However, we still cannot guarantee that different machines with different versions of Python, different gcc compilers, etc. will generate the same random numbers from the same random seed.

Reply:
I understand making the results the exact same on different machines can be challenging. My point was more that we should make the random seed more (obviously) accessible.

Reply:
Done.  I added an optional argument (`-s`, `--seed`).  By default, it choose a random seed, but users can choose their own seed.  I updated the example to use a random seed, and I updated the documentation to describe it.